### PR TITLE
Fix settings

### PR DESCRIPTION
**Pull request by geoperez**

### CODE REVIEW

**Comment by ben-ty** [comment]
@geoperez I have not tested (I could do it on monday), after reading code, some remarks :
- i think RecaptchaPrivateKey should be move to Recaptcha node below SiteKey and renamed PrivateKey, why do you have added [JsonIgnore] ?
- line 67 of Startup.cs it should be "if DEBUG" and not "if !DEBUG"
- perhaps the properties "LdapHostnames", "LdapPort", "LdapUsername", "LdapPassword" and "DefaultDomain" should be moved below the properties for the two providers. I think it will be more clear and the comment for UseAutomaticContext will be correct (it says that the credentials are below but for the moment they are above).

Thanks for the work !

It's not related to issue 221 but when I have done my testing, i noted a small issue : after changing with success a password, there id two confirmation dialog : the first is empty with juste the Ok button and the second with the right title and body. I don't know anything to angular so I can't help you but if you could take a look...

Regards

**Comment by geoperez** [comment]
@ben-ty I applied your suggestions. Let me know if it's working for you.

**Comment by geoperez** [comment]
Status?

File: README.md [hunk 1]
```diff
@@ -73,24 +74,25 @@ Use PowerShell to download and setup Passcore using the following command line,
 Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/unosquare/passcore/master/Installer.ps1'))
 ```
 
-Using the above command will install to the folder `C:\passcore` and using the HTTP Port 8080 with the default (localhost) binding.If you want to customize your installation please download the [installer script](https://raw.githubusercontent.com/unosquare/passcore/master/Installer.ps1) and the [IIS setup script](https://raw.githubusercontent.com/unosquare/passcore/master/IISSetup.ps1).
+Using the above command will install to the folder `C:\passcore` and using the HTTP Port 8080 with the default (localhost) binding.
```
Comment (jpalcala) [inline README.md]: > Using the command **shown above** will install to the folder `C:\passcore` and **use** the HTTP Port 8080 with the default (localhost) binding. 

It looks better like this

Comment (geoperez) [reply README.md]: Yes.

**Comment by ben-ty** [comment]
Hi @geoperez , i have done some adjustments (file attached) :
- Remove unused HelpTitle property and re add ChangePasswordTitle (used in change-password.html),
- Incorrect log output : log.AddConsole and log.AddDebug are already called by WebHost.CreateDefaultBuilder and it result in duplicate or mangled logs 

My tests are ok :)




[Archive.zip](https://github.com/unosquare/passcore/files/2631216/Archive.zip)

**Comment by geoperez** [comment]
I just applied your suggestions.

**Pull request merged**